### DISCUSSION TITLE
Sudden stop of analysis

### DISCUSSION THREAD

**Discussion by matthew73210** [mcguirepr89/BirdNET-Pi#761] [General]
Hey all,
I was wondering if anyone else is having this issue. Lately I've noticed that the analysis service just stops. Can't seem to pin down whats going on in the logs. The temperature is also getting higher than usual and some of the cores are nearly saturated. This didn't happen before. The temp would sit at 40 no its sits at high 50's. I used to have an issue with multi-thereading where one process would wait as it missed it's cue. Could multi threading be causing this ? I'm on an orange pi running armbian btw. 2 cores really take the load while the rest just idle by.
Cheers for your help if you have any insight.

**Comment by matthew73210** [comment]
Went back to the old model and it's calmed down.

**Reply by Fish31520** [reply]
@matthew73210
No such issue on my side so far (using a RPI 4B).

**Comment by jmtuts** [comment]
I am having what I believe to be the same issue on my Libre Le Potato.  In BirdNET -> Tools -> System Info, there is a dialogue box that say Temperature with Value (current temp) and Limit which is set to 60.0 C.  I set up a temperature monitor through SSH to watch the temps, and when it hits 60.0C BirdNET shuts down.  I am wondering if there is a way to change the Limit, 60 is just too low.  Plus I live in Arizona, US where this summer temps will average around 35C during the day

**Reply by blogdemoi** [reply]
Regarding the temp issue and the 60.0C limit, on my LePotato, I was almost at 60 for a few days and noticed that "Today's Detections" was late, 4 or 5 hours behind sometimes and that my "No Detections for Today" issue reappeared. So I put the swap at 200 and bought a case with a fan. Temperature went down to 36 and everything is fine now. Almost no more "No Detections for Today" issue, and "Today's Detections" are almost in real time.

>  I am wondering if there is a way to change the Limit, 60 is just too low. 

Could you explain what is this limit for and what will it change if the limit is higher?

Thank you.

**Comment by jmtuts** [comment]
I have ordered a heatsink and fan/box, so those should help.
What is the swap and where would I change that setting?

The Temperature Limit, I suspect, is a way to protect whatever equipment
you have running BirdNET, I was hoping to confirm with this post.
I am hoping that by raising the limit I will not run into what you
experienced with slow processing and low detections because I am getting
that and a total shutdown of the application

Regards,

**Reply by blogdemoi** [reply]
Here is a guide about the swap: https://pimylifeup.com/raspberry-pi-swap-file/ , on my LePotato I put it at 200MB. 
With the heat sink+fan and case+swap at 200 my Birdnet-Pi/LePotato is running fine.
